### ISSUE TITLE
Actions masks not applied when using callback

### ISSUE THREAD

**Issue by gowthamnatarajan** [Stable-Baselines-Team/stable-baselines3-contrib#142]
For training when I use the following it correctly applies the action masks I implemented for the MaskablePPO algorithm.

![image](https://user-images.githubusercontent.com/5133248/215952260-dba76ba9-42ad-4ef9-bdfa-e08438447038.png)

But if I use a callback, it does not use the action masks at all and samples all possible actions while training. What am I doing wrong?

![image](https://user-images.githubusercontent.com/5133248/215952676-56e3beb2-f417-4e7c-a8a7-74703b5a5e29.png)

**Comment by gowthamnatarajan** [comment]
Hi,
What additional information can I provide?

**Comment by araffin** [comment]
Just fill the issue template and provide and minimal but working example using markdown codeblock (cf. issue template).
Currently imports are missing and my guess is that's the issue (you might be using the wrong callback).

**Comment by gowthamnatarajan** [comment]
These are my import statements:

import torch
from tqdm import tqdm
from ipywidgets import FloatProgress
import os
import gym
from gym import Env
from gym.spaces import Discrete, Box
import numpy as np
import random
from stable_baselines3 import PPO
from stable_baselines3.common.vec_env import DummyVecEnv
from stable_baselines3.common.evaluation import evaluate_policy
**from stable_baselines3.common.callbacks import EvalCallback, StopTrainingOnRewardThreshold**
from time import perf_counter
from sb3_contrib import MaskablePPO
from sb3_contrib.common.maskable.utils import get_action_masks

**Comment by araffin** [comment]
> These are my import statements:

please use markdown code-block next time.

> I see that there are separate packages for Callback and evaluate_policy in sb3_contrib.

only for maskable PPO, and yes this is shown in the example from the doc.

>  What about DummyVecEnv? Is there a sb3_contrib version of that?

the VecEnv are the same.
